docs/_documentations/troubleshooting.md

```diff
@@ -461,6 +461,23 @@ Codewind displays an error. In VS Code, the error appears in the Codewind log:
 ***
 # OpenShift Do (odo) with Codewind
 
+For general information about the OpenShift Do (odo) extension on Codewind, see the [README](https://github.com/eclipse/codewind-odo-extension) file in the `codewind-odo-extension` repository.
```

Comment:
Minor tweaks:
- Change "general" to "more"
- Change "on Codewind" to "in Codewind"